### PR TITLE
Footer

### PR DESCRIPTION
**Pull request by renzobarrionuevo**
# 🧙 Description

Airfoil footer 
https://linear.app/altiplanic/issue/DAPP-59/footer

Ticket #

## ✅ Checklist

- [ ] Acceptance Criteria described in the ticket are met
- [ ] Texts and strings are in the related i18n file
- [ ] Utilities have unit tests
- [ ] User stories and functionalities have integration or e2e tests
- [ ] If adding a new token or network
  - [ ] Registry package is updated
  - [ ] Asset images are added
- [ ] Related packages.json are upgraded
- [ ] Changelog is updated

### CODE REVIEW

**Review by renzobarrionuevo — COMMENTED** [review]
test

File: CHANGELOG.md [hunk 1]
```diff
@@ -36,6 +36,7 @@ Ref: https://keepachangelog.com/en/1.0.0/
 
 ## Unreleased
 
+-(fix) DAPP-59 | packages/i18n 1.0.6 apps/mission 1.0.36 | footer airfoil
```
Comment (julia-script) [inline CHANGELOG.md:39]: not sure, but it might not be necessary to bump versions until this version is released
opinions?

**Review by julia-script — APPROVED** [review]
looking good

File: CHANGELOG.md (hunk 1 above)
Comment (renzobarrionuevo) [reply CHANGELOG.md:39]: Ok I was doing it to all the PRs but I remove it.

Comment (julia-script) [reply CHANGELOG.md:39]: no need to remove it..I just think it's not worth the hassle :)

**Pull request merged**